models.py

```diff
@@ -0,0 +1,80 @@
+from typing import List
+from datetime import datetime
+
+
+class Professor:
+    
+    def __init__(self, trigramm: str, first_name: str = "", last_name: str = ""):
+        """_summary_
+
+        Args:
+            trigramm (str): Prof's trigramm (ex: "JOE")
+            first_name (str, optional): Prof's first name (ex: "John"). Defaults to "".
+            last_name (str, optional): Prof's last name (ex: "Doe"). Defaults to "".
+        """
+        self.trigramm = trigramm
+        self.first_name = first_name
+        self.last_name = last_name
+        self.courses = []
```

Comment:
Shouldn't be a list of course ? If yes, please, remember to type it with `List["Course"]`